### DISCUSSION TITLE
How to customize selected time styles in Mantine TimePicker?

### DISCUSSION THREAD

**Discussion by H4kim** [mantinedev/mantine#6663] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

6.0.22

### What package has an issue?

@mantine/dates

### What framework do you use?

create-react-app (CRA)

### In which browsers you can reproduce the issue?

All

### Describe the bug

I trying to customize the default style for the selected time, As you can see, the default colour for the selected time is Blue, and i want it to be Red.

By looking at [mantine's documentation](https://by%20looking%20at%20[the%20doc][2],%20i%20can't%20find%20any/), i can't find anything related to the picker

<img width="277" alt="image" src="https://github.com/user-attachments/assets/25a399b2-61e8-4661-9830-5393a4f1c228">

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
The UI displayed on the screenshot is a native browser time picker, it is not part of Mantine. To learn how to customize it, see https://stackoverflow.com/questions/53544872/how-to-style-the-input-type-time

**Reply by blur94** [reply]
Hi @rtivital, The solution offered in the SOF link didn't work. Please, has there been any fix since this issue was raised? 

Note: I'll be glad to get a solution that's compatible with v6.

**Comment by blur94** [comment]
Hi @H4kim,
I'm having the same issue. Have you gotten a fix for this? 

Note: My project is based on mantine v6.
